app/code/Magento/Sales/Test/Mftf/Test/AdminOpenCreditmemoViewPageWithWrongCreditmemoIdTest.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminOpenCreditmemoViewPageWithWrongCreditmemoIdTest">
+        <annotations>
+            <stories value="Creditmemo Page With Wrong Creditmemo Id"/>
+            <title value="Open Creditmemo View Page with Wrong Creditmemo Id"/>
+            <description value="Open Creditmemo View Page with Wrong Creditmemo Id."/>
+            <severity value="MAJOR"/>
+            <group value="sales"/>
+        </annotations>
+
+        <before>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="LoginAsAdmin"/>
+        </before>
+
+        <after>
+            <actionGroup ref="AdminLogoutActionGroup" stepKey="logout"/>
+        </after>
+
+        <actionGroup ref="AdminGoToCreditmemoViewActionGroup" stepKey="navigateOpenCreditmemoViewPage">
+            <argument name="identifier" value="test"/>
+        </actionGroup>
+
+        <waitForPageLoad stepKey="waitForPageLoad"/>
+
+        <seeInCurrentUrl url="{{AdminCreditMemosGridPage.url}}" stepKey="redirectToCreditMemosGridPage"/>
+
+        <actionGroup ref="AssertAdminPageIs404ActionGroup" stepKey="see404PageOnAdmin"/>
```

Comment:
Shouldn't we have the same as the other test here `seeErrorMessage` ? If the redirect is successful, I don't think the user gets 404.